src/components/Tooltip/TooltipHost.Props.ts

```diff
@@ -1,25 +1,32 @@
 import * as React from 'react';
-import { Tooltip } from './Tooltip';
-import { ITooltipProps } from '../../index';
+import { TooltipHost } from './TooltipHost';
+import { ITooltipProps, TooltipDelay } from '../../index';
+import { DirectionalHint } from '../../common/DirectionalHint';
 
 /**
  * Tooltip component props.
  */
 
-export interface ITooltipHostProps extends React.Props<Tooltip> {
+export interface ITooltipHostProps extends React.HTMLProps<HTMLDivElement | TooltipHost> {
   /**
    * Properties to pass through for Callout, reference detail properties in ITooltipProps
    */
   calloutProps?: ITooltipProps;
 
   /**
-   * Number of milliseconds to wait before showing the tooltip
-   * @default 1000
+   * Length of delay
+   * @default medium
    */
-  delay?: number;
+  delay?: TooltipDelay;
```

Comment:
To clarify. This is a really new component, so the breaking change is probably ok. However, we would really want to clarify when to use which delay. E.g. when do i use 'long'? What scenarios would I have no delay?

Also it somewhat goes against the nuances i was showing you earlier. If the user hovers over something, there's a delay. If they quickly hover over something else, there's no delay. So perusing through the command bar tooltips is easy if that's your intent.